### ISSUE TITLE
webrtc-voicemail  throws a stack trace

### ISSUE THREAD

**Issue by la3lma** [pion/webrtc-voicemail#2]
### Your environment.
  - Version: f38a3673149c32d6de87b210b558162b0ca85cc0
  - Browser:  Google Chrome Version 81.0.4044.129 (Official Build) (64-bit), MacBook Pro 2018, macOS 10.15.3
  - Other Information - *stacktraces, related issues, suggestions how to fix, links for us to have context*

### What did you do?

1.  Ran the program from goland (stack trace included) and also from command line (identical behaviour).
2.  Pushed the button to start recording
3.  Observed that the .ogg file continued to grow until I closed the browser window.   The actual audio recorded is of good quality, so no issues there.
4.  Observed stack trace  (see below)
5.  In goland, looked at line 60 and saw that AddTranceiver is now a deprecated method (may not be relevant) 

### What did you expect?

1. Some kind of response that  a recording had taken place.
2. Some way to stop to the recording.
3. No stack traces thrown during execution.
4. No use of deprecated functions in demonstration code.

### What happened?

```
2020/05/02 11:53:22 http: panic serving [::1]:50314: sdp: invalid syntax ``
goroutine 10 [running]:
net/http.(*conn).serve.func1(0xc00015e320)
        /usr/local/Cellar/go/1.13/libexec/src/net/http/server.go:1767 +0x139
panic(0x1418da0, 0xc000123990)
        /usr/local/Cellar/go/1.13/libexec/src/runtime/panic.go:679 +0x1b2
main.createVoicemail(0x15464e0, 0xc0002661c0, 0xc0003ba200)
        /Users/rmz/git/webrtc-voicemail/main.go:65 +0x749
net/http.HandlerFunc.ServeHTTP(0x14ccbf0, 0x15464e0, 0xc0002661c0, 0xc0003ba200)
        /usr/local/Cellar/go/1.13/libexec/src/net/http/server.go:2007 +0x44
net/http.(*ServeMux).ServeHTTP(0x1803c60, 0x15464e0, 0xc0002661c0, 0xc0003ba200)
        /usr/local/Cellar/go/1.13/libexec/src/net/http/server.go:2387 +0x1bd
net/http.serverHandler.ServeHTTP(0xc000134000, 0x15464e0, 0xc0002661c0, 0xc0003ba200)
        /usr/local/Cellar/go/1.13/libexec/src/net/http/server.go:2802 +0xa4
net/http.(*conn).serve(0xc00015e320, 0x1547260, 0xc0002e4300)
        /usr/local/Cellar/go/1.13/libexec/src/net/http/server.go:1890 +0x875
created by net/http.(*Server).Serve
        /usr/local/Cellar/go/1.13/libexec/src/net/http/server.go:2927 +0x38e

```

**Comment by Sean-Der** [comment]
Hey @la3lma 

Thanks for the PR, sorry I have been busy! I would love to expand this repo and make it more robust/featureful. I made it as a demo for a talk I gave, and then never got around to finishing it more :(

If you are interested would love to work together on this. If you have a list of things to fix I can knock them out (and merge your PRs!)

**Comment by wasauce** [comment]
@Sean-Der - is there a place I can see that talk? Sounds interesting!

**Comment by Sean-Der** [comment]
@wasauce https://vimeo.com/380076363

If you have any questions I am in [Slack](https://pion.ly/slack). It has been a while so certain things might be out of date, ideas/motivation have stayed the same :)

**Comment by wasauce** [comment]
@Sean-Der - thanks very much. Great video!

**Comment by Sean-Der** [comment]
The deprecated method usage was fixed in 4755a8c835cc46fd51a9c6ea107c18473165b425

re: suggested features. I don't think should add more. My hope is that repo demonstrates things people can/could build, and adding more code would make it harder for people to copy and modify.

If anyone disagrees please re-open and would love to keep iterating!
